### PR TITLE
BOM-74

### PR DESCRIPTION
**Pull request by awais786**
https://openedx.atlassian.net/browse/BOM-74

Update the version.

### CODE REVIEW

**Comment by awais786** [comment]
@feanil every thing is already updated.
This previous [PR-52](https://github.com/edx/edx-analytics-data-api-client/pull/52) updated the stuff.

File: setup.py [hunk 1]
```diff
@@ -9,6 +11,19 @@
     url='https://github.com/edx/edx-analytics-data-api-client',
     description='Client used to access edX analytics data warehouse',
     long_description=open('README.rst').read(),
+    license='Apache License 2.0',
```
Comment (awais786) [inline setup.py]: @feanil Check this license

File: analyticsclient/__init__.py [hunk 2]
```diff
@@ -1 +1 @@
-__version__ = '0.15.3'
+__version__ = '0.15.4'
```
Comment (Ayub-Khan) [inline analyticsclient/__init__.py]: @feanil  As it was already updated to python3 do we still need to release a new version.

**Comment by awais786** [comment]
@feanil check again. I will release after merging this PR.

**Pull request merged**